src/graph/impl/KokkosGraph_Distance1Color_impl.hpp

```diff
@@ -3287,62 +3201,36 @@ class GraphColor_EB:public GraphColor <HandleType,in_row_index_view_type_,in_non
     }
   };
 
-
   /**
-   * \brief Functor to perform parallel prefix sum for edges so that the position
-   * on the next conflictlist is calculated.
+   * \brief Functor to create the new work array with a parallel prefix sum.
    */
-  struct parallel_prefix_sum{
-    size_type_temp_work_view_t _edge_conflict_indices;
-    char_temp_work_view_type _edge_conflict_marker;
-    size_type_temp_work_view_t _pps_view;
-
-    parallel_prefix_sum(
-        size_type_temp_work_view_t edge_conflict_indices,
-        char_temp_work_view_type edge_conflict_marker,
-        size_type_temp_work_view_t pps_view):
-          _edge_conflict_indices(edge_conflict_indices),
-          _edge_conflict_marker(edge_conflict_marker),
-          _pps_view(pps_view){}
+  struct ppsWorklistFunctorEB {
```

Comment:
I think what I assumed above was correct, based on this functor. Looks like we have two PPS functors now.